### PR TITLE
COV-44: build sign up page

### PR DESCRIPTION
**Pull request by jason-gerard**
JIRA: https://soen-390-covid-tracker.atlassian.net/browse/COV-42

## Changes
- Add sign up form
- Connect form with API
- Update routes to follow standard naming conventions
- Extra file cleanup

### CODE REVIEW

File: client/src/@core/components/wizard/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // ** React Imports
```
Comment (jason-gerard) [inline client/src/@core/components/wizard/index.js:1]: Is this file supposed to be changed?

File: client/src/views/forms/wizard/WizardHorizontal.js [hunk 2]
```diff
@@ -18,36 +18,27 @@ const WizardHorizontal = () => {
     const [stepper, setStepper] = useState(null);
```
Comment (jason-gerard) [inline client/src/views/forms/wizard/WizardHorizontal.js]: This one too?

File: client/src/@core/components/wizard/index.js (hunk 1 above)
Comment (ejazali-rezayi) [reply client/src/@core/components/wizard/index.js:1]: I only added the logo in this file, I tried to add it in other files, but this one had the best positioning for the logo, I think it should be fine.

File: client/src/views/forms/wizard/WizardHorizontal.js (hunk 2 above)
Comment (ejazali-rezayi) [reply client/src/views/forms/wizard/WizardHorizontal.js]: This one I will rollback the changes.

File: client/src/views/forms/wizard/steps-with-validation/PersonalSignUp.js [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+    const [data, setData] = useState(null);
+
+    const signUpSchema = yup
+        .object()
+        .shape({
+            firstName: yup
+                .string()
+                .required("Enter a first name.")
+                .matches(/^[aA-zZ\s]+$/, "First name must only contain alphabetic characters.")
+                .max(32, "First name must be 32 characters or less."),
```
Comment (Andre-Ibrahim) [inline client/src/views/forms/wizard/steps-with-validation/PersonalSignUp.js:36]: The max in our server side validation is 25

File: client/src/views/forms/wizard/steps-with-validation/PersonalSignUp.js [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+            lastName: yup
+                .string()
+                .required("Enter a last name.")
+                .matches(/^[aA-zZ\s]+$/, "Last name must only contain alphabetic characters.")
+                .max(32, "Last name must be 32 characters or less."),
```
Comment (Andre-Ibrahim) [inline client/src/views/forms/wizard/steps-with-validation/PersonalSignUp.js:41]: same as above

**Review by Andre-Ibrahim — APPROVED** [review]

File: client/src/views/forms/wizard/steps-with-validation/PersonalSignUp.js (hunk 3 above)
Comment (jason-gerard) [reply client/src/views/forms/wizard/steps-with-validation/PersonalSignUp.js:36]: Noted, can fix in a follow up task

File: client/src/views/forms/wizard/steps-with-validation/PersonalSignUp.js (hunk 4 above)
Comment (jason-gerard) [reply client/src/views/forms/wizard/steps-with-validation/PersonalSignUp.js:41]: Noted, can fix in a follow up task

**Pull request merged**